compiler/quilt/tools/package.py

```diff
@@ -74,6 +74,33 @@ def __init__(self, store, user, package, path, contents=None, pkghash=None):
 
         self._contents = contents
 
+    def __getitem__(self, item):
+        node = self.get_contents()
+
+        if '/' in item and '.' in item:
```

Comment:
Let's pick just one syntax for accessing package components by path. My vote is to use '.' as the separator.

Reply:
I'm rather for '/', myself.  Originally, I was for the dot notation, but these considerations changed my thoughts on it:

* Our `user/pkg/sub/node/notation` uses slashes, and this is consistent with that.
* Using slashes leaves open the option of relaxing node names to be more like filenames in the future
* It would allow us to use a pathlib.PurePosixPath object when doing common pathlike operations.

Also, as we move toward using item['notation'] for nodes, looking like a Python identifier becomes less important.